presto-docs/src/main/sphinx/security/ldap.rst

```diff
@@ -0,0 +1,212 @@
+===================
+LDAP Authentication
+===================
+
+Presto can be configured to enable frontend LDAP authentication over
+HTTPS for clients, such as the :ref:`cli_ldap`, or the JDBC and ODBC
```

Comment:
hmm I copied this from the kerberos documentation, and there we add drivers too.